templates/web-application/.prettierignore

```diff
@@ -0,0 +1,2 @@
+*
+!**/*.css
```

Comment:
I add `!**/*.css` here vs https://github.com/patricklafrance/pnpm-ts5-swc-esm-jest/blob/master/.eslintrc.json as recommended in https://github.com/workleap/wl-web-configs/tree/main/packages/stylelint-config#prettier